build_container.sh

```diff
@@ -26,7 +26,7 @@ if [ $? -eq 0 ]; then
     attr -s test -V test $testfile > /dev/null
     if [ $? -ne 0 ]; then
         echo "ERROR: $EESSI_TMPDIR does not support extended attributes!" >&2
-       exit 2
+        #exit 2
```

Comment:
Rather than commenting this out, we should just drop this `attr` check, should be OK thanks to https://github.com/EESSI/filesystem-layer/pull/125

I'll do that in a follow-up PR

Reply:
see #189